content/pages/home.json

```
@@ -8,8 +8,8 @@
       "_template": "navbar",
       "banner": {
         "display": false,
```

```suggestion
        "display": true,
```

I guess you have to set this to true for the banner to show